packages/meteor/src/MeteorAction.ts

```diff
@@ -0,0 +1,121 @@
+import { Action, ActionImpl, ActionOptions, BindOptions, ValueOrFunction } from '@moxb/moxb';
+import { MeteorMethodControl } from './MeteorMethod';
+import { Bit, BitImpl } from './Bit';
+
+/**
+ * The main idea here is that it should be trivial to define a (moxb) Action that is backed by a Meteor method,
+ * so that we have type safety on the input and on the output side, we can't misspell the method name, and
+ * we also get automatic integration for things like indicating the pending state on buttons, etc.
+ */
+
+/**
+ * This is how we describe a Meteor method-based action
+ */
+interface MeteorActionOptions<Input, Output> extends BindOptions {
+    /**
+     * A (control object to a) Meteor method to use
+     */
+    method: MeteorMethodControl<Input, Output>;
+
+    /**
+     * Do we want to see debug output related to this action?
+     */
+    debug?: boolean;
+
+    /**
+     * The input to pass to this method (or a function to collect it)
+     */
+    input: ValueOrFunction<Input>;
+
+    /**
+     * An optional callback to call (with the current input) before actually executing the method call
+     * @param input
+     */
+    before?: (input: Input) => void;
+
+    /**
+     * Error handler to call when the method fails
+     */
+    failed: (error: Meteor.Error) => void;
+
+    /**
+     * A callback to call when the method succeeds
+     */
+    returned: (result: Output) => void;
+}
+
+/**
+ * This function will create a Meteor method-backed moxb Action based on the provided description (see above).
+ */
+export function createMeteorAction<Input, Output>(options: MeteorActionOptions<Input, Output>): Action {
+    const { method, input, before, failed, returned, debug, ...rest } = options;
+
+    const _pending: Bit = new BitImpl();
```

Comment:
@csillag Why do you use a new single bit datastore here, and not a simple boolean? What are the benefits?

Reply:
> Why do you use a new single bit datastore here, and not a simple boolean? What are the benefits?

There were no benefits. I have simplified the code in 441d7a414fc0ee5a3233d60ed578745e3dca8ece. 

The reason why I did it that way in the original implementation was that `createMeteorAction` is a function, not a class, and one can't use the `@observable` decorator on simple variables. I was in a hurry, and didn't want to investigate this, so I made a (minimal) class for handling the reactivity.

But now I implemented in properly, using `observable.box`.